server/src/main/java/io/crate/metadata/sys/SysAllocationsTableInfo.java

```diff
@@ -55,5 +56,6 @@ public class SysAllocationsTableInfo {
             ColumnIdent.of("partition_ident"),
             ColumnIdent.of("shard_id")
         )
+        .withRouting((state, routingProvider, sessionSettings) -> Routing.forTableOnMasterNode(IDENT, state.nodes()))
```

Comment:
```suggestion
        .withRouting((state, routingProvider, sessionSettings) -> Routing.forTableOnSingleNode(IDENT, state.nodes().getMasterNodeId()))
```

Doesn't seem worth an extra method?